toqito/states/mutually_unbiased_basis.py

```diff
@@ -26,6 +26,13 @@ def mutually_unbiased_basis(dim: int) -> list[np.ndarray]:
         M_1 = \left\{\frac{|0\rangle + |1\rangle}{\sqrt{2}}, \frac{|0\rangle - |1\rangle}{\sqrt{2}}\right\}
         M_2 = \left\{\frac{|0\rangle + i|1\rangle}{\sqrt{2}}, \frac{|0\rangle - i|1\rangle}{\sqrt{2}}\right\}
 
+    >>> from toqito.matrix_ops import mutually_unbiased_basis
```

Comment:
```suggestion
    >>> from toqito.states import mutually_unbiased_basis
```
The import path was incorrect